review-tools/gitaddrev

```diff
@@ -151,6 +151,12 @@ foreach (@ARGV) {
         $min_authors = 2;
         $min_otc = 0;
         $min_omc = 0;
+    } elsif (/--installer$/) {
+        $WHAT = 'installer';
+        $min_authors = 2;
```

Comment:
A `min_authors` value of 2 is meaningless. Well actually it means the same as 1. Really the only meaningful settings are 0 and 1. 0 means the author does not count as a reviewer. 1 means the author does count. I think this variable is misnamed (it's not a minimum of anything).

I would really expect the installer repo to have the same review criteria as the main source repo (i.e. the author does not count as a reviewer).

I note `perftools` also has a `min_authors` value of 2, which is equally strange IMO.

Reply:
fixed both to be one